contracts/treasury/Collector.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.8.10;
+
+import {IERC20} from '@aave/core-v3/contracts/dependencies/openzeppelin/contracts/IERC20.sol';
+import {ICollector} from './interfaces/ICollector.sol';
+import {VersionedInitializable} from '@aave/core-v3/contracts/protocol/libraries/aave-upgradeability/VersionedInitializable.sol';
+
+/**
+ * @title Collector
+ * @notice Stores the fees collected by the protocol and allows the fund administrator to approve or transfer the collected ERC20 tokens.
+ * @author Aave
+ **/
+contract Collector is ICollector, VersionedInitializable {
```

Comment:
Would swap the inheritance order (for consistency)
`is VersionedInitializable, ICollector`